release/platform/platform.go

```diff
@@ -12,180 +22,185 @@ import (
 type Platform struct {
 	Name string
 	Arch string
+	OS   string
+	Pkg  string
+}
```

Comment:
as much as possible, trying to move data about platforms into the structs themselves instead of having it in helper funcs that switch on the name/arch fields

Reply:
I'm wondering if we should stick DebArch in here too, but I think that's probably overkill, since not every platform needs that.